src/Conf.ml

```diff
@@ -1872,9 +1866,9 @@ let is_in_listing_file ~quiet ~listings ~filename =
       try
         In_channel.with_file (Fpath.to_string listing_file) ~f:(fun ch ->
             let lines =
-              In_channel.input_lines ch
-              |> List.mapi ~f:(fun i s -> (i + 1, String.strip s))
-              |> List.filter ~f:(fun (_, l) -> not (drop_line l))
+              List.filter_mapi (In_channel.input_lines ch) ~f:(fun i s ->
+                  let l = String.strip s in
+                  Option.some_if (not (drop_line l)) (i + 1, l))
```

Comment:
`Option.some_if` is only 6 characters shorter than an `if then Some ... else None` so I'm not sure it's worth the complexity. We are using it a lot in OCamlformat so it's OK.

Reply:
done